app/views/profiles/show.html.erb

```diff
@@ -4,7 +4,7 @@
       <div id="avatar-frame">
         <%=
           image_tag(
-            @user.gravatar_url(size: 300, secure: request.ssl?).html_safe,
+            @user.gravatar_url(size: 300, secure: true).html_safe,
```

Comment:
Would it be better to migrate this to `gravatar` helper call instead? That way we can control this from one place. Also feel free to check if there is any other call to `@user.gravatar_url` and migrate that to `gravatar` helper call)

Reply:
@simi I pushed a version for the helper to accept options, WDYT?

Reply:
Looks great!